SofaKernel/modules/SofaSimpleFem/src/SofaSimpleFem/HexahedronFEMForceField.inl

```diff
@@ -124,14 +127,16 @@ void HexahedronFEMForceField<DataTypes>::init()
     }
 
 
-    if( m_topology->getNbHexahedra()<=0 )
+    if ( m_topology->getNbHexahedra() <= 0 )
```

Comment:
same as a recent PR. It is better to check the type of element in tricky case when you would like to start the simulation with empty mesh.

```suggestion
    if ( m_topology->getTopologyType() != sofa::core::topology::TopologyElementType::HEXAHEDRON )
```

Reply:
Does it mean topologies with multiple types of elements cannot be supported?

Reply:
Ok @fredroy told me this is not working. So just revert my suggestion and we will see this point later.